app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -29,6 +29,21 @@ def failure
     end
   end
 
+  def sign_out
+    case current_journey_routing_name
+    when "further-education-payments-provider"
+      redirect_to(
+        claim_path(
+          journey: current_journey_routing_name,
+          slug: "sign-in"
+        ),
+        notice: "You've been signed out"
+      )
```

Comment:
I think we probably want to clear the session here. We can do something like this

```suggestion
      claim = journey_sessions.answers.claim
      clear_journey_sessions!
      redirect_to(
        Journeys::FurtherEducationPayments::Provider::SlugSequence.verify_claim_url(claim),
        notice: "You've been signed out"
      )
```
So the claim they're trying to access is in the url and they'll be able to sign in again and start a new session